orion-ui/src/router/index.ts

```diff
@@ -11,22 +11,32 @@ const routeRecords: AppRouteRecord[] = [
     redirect: routes.flowRuns(),
   },
   {
-    name: 'flow-runs',
```

Comment:
I wonder if we could just take this section out?  Any ideas if we need to keep it for anything @pleek91?

Reply:
I only put this here in case someone has the page "starred" but they can always reaccess the page by clicking!